tests/expected/matlab/MyFactorPosePoint2.m

```diff
@@ -29,6 +32,16 @@ function delete(obj)
     %DISPLAY Calls print on the object
     function disp(obj), obj.display; end
     %DISP Calls print on the object
+    function varargout = print(this, varargin)
+      % PRINT usage: print(string s, KeyFormatter keyFormatter) : returns void
+      % Doxygen can be found at https://gtsam.org/doxygen/
+      if length(varargin) == 2 && isa(varargin{1},'char') && isa(varargin{2},'gtsam.KeyFormatter')
```

Comment:
It's interesting that the print method is here. I think by default it should use `obj.display`, but maybe that does not support custom KeyFormatter.

Reply:
I am not even touching the Matlab code right now because I am not the right person to do that.